src/borg/archiver.py

```diff
@@ -1233,11 +1233,14 @@ def do_break_lock(self, args, repository):
 
         {now}
 
-            The current local date and time.
+            The current local date and time, by default in ISO-8601 format.
+            You can also supply your own format string, e.g. {now:%y}
```

Comment:
The bigger problem might be that you could only do one backup per year with a name like that 😜 

I'll update that with more reasonable examples.